mkDocs.sh

```diff
@@ -53,7 +45,35 @@ docker run --rm \
 	--network arlas_default \
 	byrnedo/alpine-curl \
     -c 'i=1; until curl -XGET http://arlas-server:9999/arlas/swagger.yaml -o /opt/maven/target/tmp/swagger.yaml; do if [ $i -lt 30 ]; then sleep 1; else break; fi; i=$(($i + 1)); done'
+    
+echo "=> Generate API"
+docker run --rm \
+	-v $PWD:/opt/gen \
+	-v $HOME/.m2:/root/.m2 \
+	gisaia/swagger-codegen:2.3.1
+	
+echo "=> Generate documentation of API typescript client"
```

Comment:
Change to `Generate Typescript client documentation`

Reply:
ok